components/camera/replaypcd/replaypcd.go

```diff
@@ -247,12 +347,20 @@ func (replay *pcdCamera) Reconfigure(ctx context.Context, deps resource.Dependen
 		}
 	}
 
+	if replayCamConfig.BatchSize == nil {
+		replay.limit = 1
+	} else {
+		replay.limit = *replayCamConfig.BatchSize
```

Comment:
is zero fine?

Reply:
I updated Validate; I assume Validate will always be called before Reconfigure and I don't need to check again here? Or do I assume wrong

Reply:
yep thats fine